src/scripts.js

```diff
@@ -1,168 +1,213 @@
-import './css/base.scss';
-import './css/styles.scss';
+// import './css/base.scss';
+// import './css/styles.scss';
 
-import recipeData from './data/recipes';
-import ingredientData from './data/ingredients';
-import users from './data/users';
+// import recipeData from './data/recipes';
```

Comment:
I think having the `scripts` in `index.html` is fine but we'll have to use `import/export` syntax later.